fluent-react/src/localized.js

```diff
@@ -116,12 +100,36 @@ export default class Localized extends Component {
     }
 
     const msg = mcx.getMessage(id);
-    const args = toArguments(this.props);
-    const { parts, attrs } = l10n.formatCompound(mcx, msg, args);
+    const [args, elems] = toArguments(this.props);
+    const { value, attrs } = l10n.formatCompound(mcx, msg, args);
+
+    if (value === null || !value.includes('<')) {
```

Comment:
I suppose there are no unexpected side-effects if the string inlcudes the `<` character outside of the context of a markup element. For example `The result is that 0 < 3 ...`.

It looks like the `parseMarkup` that comes next will just leverage the DOM API to split out the different parts, and so this step is more of an optimization in case the translation string does not contain react components. Just thinking out loud and checking my understanding - does that sound about right?

Reply:
Correct. In the rare scenario of a false-positive (like in the example you gave) we'll pay a bit extra to parse HTML which is not there. It will parse as a text node and things should work just fine. I'll add a test.